frog_api/views/data.py

```diff
@@ -85,16 +93,16 @@ class VersionDataView(APIView):
     """
 
     @staticmethod
-    def write_new_entries(
+    def create_entries(
         req_data: dict[str, Any], project_slug: str, version_slug: str
     ) -> int:
         project = get_project(project_slug)
         version = get_version(version_slug, project)
 
         if "api_key" not in req_data:
             raise MissingAPIKeyException()
```

Comment:
Only other thing I can see is

Reply:
I think this is better for the serializer validation